src/scripts/smokebomb.js

```diff
@@ -22,7 +22,7 @@ var images = [
     "http://media.fyre.co/ehIL2qumSGeNTEzSqK3X_ninja_magic.gif",
     "http://i.imgur.com/hPip3nF.gif",
     "http://cdn.makeagif.com/media/10-16-2015/Bxsvl2.gif",
-    "http://i.imgur.com/DPhNOpo.gifv"
```

Comment:
i can say the same about this update

![anders slam](https://cloud.githubusercontent.com/assets/2065840/14788761/78c07834-0ad7-11e6-979a-7e8fc4f56683.gif)